apps/frontend/src/libs/hooks/use-popover-position/use-popover-position.hook.ts

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "~/libs/hooks/hooks.js";
+
+type Properties = {
+	top?: number;
+};
+
+const usePopoverPosition = <T extends HTMLElement>(
+	reference: React.RefObject<T>,
+	isOpen: boolean,
+	hasPositioning: boolean = false,
+): Properties => {
+	const [position, setPosition] = useState<Properties>({});
+
+	useEffect(() => {
+		if (!hasPositioning || !isOpen || !reference.current) {
+			setPosition({});
+
+			return;
+		}
```

Comment:
I think we don't need to change positioning depending on isOpen (we will just not display it).
```suggestion
		if (!hasPositioning || !reference.current) {
			return;
		}
```